boards/beagle/pocketbeagle_2/doc/index.rst

```diff
@@ -119,3 +172,6 @@ References
 
 .. _BeagleBoard Imaging Utility:
    https://github.com/beagleboard/bb-imager-rs/releases
```

Comment:
Let's try to get a more official home for this. Let's replace the information at https://www.beagleboard.org/bb-imager.

Reply:
I do not have access to change that page, so sure, we can update the link once that happens.